deploy/provider-gcp-plugin.yaml

```diff
@@ -68,6 +68,9 @@ spec:
         app: csi-secrets-store-provider-gcp
     spec:
       serviceAccountName: secrets-store-csi-driver-provider-gcp
+      hostNetwork: false
+      hostPID: false
+      hostIPC: false
```

Comment:
we probably want to add these changes to charts/secrets-store-csi-driver-provider-gcp/templates/daemonset.yaml as well at some point

Reply:
I have raised a new PR for the Helm DaemonSet changes